### PR TITLE
Feature/dashboard sync

### LINKED ISSUE #11: FE As a user, I should be able to see my submitted mood and habits for the day in both my dashboard and form view.

### PR DESCRIPTION
**Pull request by woaishenzhen**
### Features
- Combine mood form and habit form on one page
- Display updated mood and habits in dashboard
- When navigated from dashboard to form page, display the same mood and habit status as dashboard
- When user updates habits on form page and goes to dashboard, dashboard displays updated habits

### Refactors 
- Move some local states and functions to context

### Issues and Technical Debt
- 

### Other Comments, Concerns, or Questions
- closes #11 #12

### Checklist
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature work
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/MoodForm/MoodForm.tsx [hunk 1]
```diff
@@ -1,51 +1,54 @@
...
   return (
     <section className="mood-form-container">
-      { !isSubmitted ?
-        <form className="mood-form" onSubmit={ handleSubmit }>
-          <h2>
-            How are you feeling today?
-          </h2>
+      {todaysMood ? (
+        <div>
+          <p>I am feeling: {displayMood(todaysMood!.mood)}</p>
```
Comment (nzambonivergara) [inline src/components/MoodForm/MoodForm.tsx]: We can make this piece a component since we reuse this in both MoodForm and Dashboard.

Comment (TannerDale) [reply src/components/MoodForm/MoodForm.tsx]: This would be a good change to make

File: src/utils/context.tsx [hunk 2]
```diff
@@ -1,46 +1,56 @@
...
-  // const [habitRecorded, setHabitRecorded] = useState<boolean>(false)
+  const [todaysMood, setTodaysMood] = useState<Mood | null>(null)
+
+  const [todaysHabits, setTodaysHabits] = useState<Habit[]>([])
 
-  // const [todaysMood, setTodaysMood] = useState<Mood | null>(null)
+  const displayMood = (mood: number) => {
+    switch (mood) {
+      case 0:
+        return "😭"
```
Comment (TannerDale) [inline src/utils/context.tsx:40]: I still get a laugh seeing emojis in code

**Review by TannerDale — CHANGES_REQUESTED** [review]

**Comment by beizy** [comment]
WTF the checks just passed?

**Review by nzambonivergara — COMMENTED** [review]
Great job implementing this! It was not an easy task. I think once we make that component and add some styling we can merge. :)

File: src/components/MoodForm/MoodForm.tsx (hunk 1 above)
Comment (woaishenzhen) [reply src/components/MoodForm/MoodForm.tsx]: New component added

File: src/utils/context.tsx (hunk 2 above)
Comment (woaishenzhen) [reply src/utils/context.tsx:40]: Good for you! 😉

**Pull request merged**